doc/source/news.rst

```diff
@@ -5,6 +5,653 @@
+
+First of all, we introduce the main changes in 13.0.0.
+Then, we introduce the hilight and summary of changes from Groonga 12.0.0 to 12.1.2.
+
+New Features and Improvements in 13.0.0
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Improvements
+^^^^^^^^^^^^
+
+* [:doc:`/reference/normalizers`] Added a new Normalizer ``NormalizerNFKC150`` based on Unicode NFKC (Normalization Form Compatibility Composition) for Unicode 15.0.
+
+* [:doc:`/reference/token_filters`] Added a new TokenFilter ``TokenFilterNFKC150`` based on Unicode NFKC (Normalization Form Compatibility Composition) for Unicode 15.0.
+
+* [:doc:`reference/normalizers/normalizer_nfkc150`] Added a new option ``unify_katakana_g_sounds``.
+
+  We can normalize "グァ -> ガ", "グィ -> ギ", "グェ -> ゲ", and "グォ -> ゴ" with this option.
+
+  .. code-block::
+
+     normalize \
+       'NormalizerNFKC130("unify_katakana_g_sounds", true, \
+                          "report_source_offset", true)' \
+       "グァグィグェグォ" \
+       WITH_CHECKS|WITH_TYPES
+
+     [
+       [
+         0,
+         0,
+         0
+       ],
+       {
+         "normalized":"ガギゲゴ",
+         "types":["katakana","katakana","katakana","katakana"],
+         "checks":[6,0,0,6,0,0,6,0,0,6,0,0],
+         "offsets":[0,6,12,18]
+       }
+     ]
+
+* [:doc:`reference/normalizers/normalizer_nfkc150`] Added a new option ``unify_katakana_di_sound``.
+
+  We can normalize "ヂ -> ジ" with this option.
+
+  .. code-block::
+
+     normalize \
+       'NormalizerNFKC130("unify_katakana_di_sound", true, \
+                          "report_source_offset", true)' \
+       "ヂ" \
+       WITH_CHECKS|WITH_TYPES
+     [
+       [
+         0,
+         0,
+         0
+       ],
+       {
+         "normalized":"ジ",
+         "types":["katakana"],
+         "checks":[3,0,0],
+         "offsets":[0]
+       }
+     ]
+
+* [:doc:`reference/normalizers/normalizer_nfkc150`] Added a new option ``unify_katakana_wo_sound``.
+
+  We can normalize "ヲ -> オ" with this option.
+
+  .. code-block::
+
+     normalize \
+       'NormalizerNFKC130("unify_katakana_wo_sound", true, \
+                          "report_source_offset", true)' \
+       "ヲ" \
+       WITH_CHECKS|WITH_TYPES
+     [
+       [
+         0,
+         0,
+         0
+       ],
+       {
+         "normalized":"オ",
+         "types":["katakana"],
+         "checks":[3,0,0],
+         "offsets":[0]
+       }
+     ]
+
+* [:doc:`reference/normalizers/normalizer_nfkc150`] Added a new option ``unify_katakana_z_sound``.
+
+  We can normalize "ズァ -> ザ", "ズィ -> ジ", "ズェ -> ゼ", and "ズォ -> ゾ" with this option.
+
+  .. code-block::
+
+     normalize \
+       'NormalizerNFKC130("unify_katakana_z_sounds", true, \
+                          "report_source_offset", true)' \
+       "ズァズィズェズォ" \
+       WITH_CHECKS|WITH_TYPES
+     [
+       [
+         0,
+         0,
+         0
+       ],
+       {
+         "normalized":"ズァズィズェズォ",
```

Comment:
The normalized value seems wrong.
Would you update the sample code?
(The option name also need to be updated.)